tests/baselines/reference/tsserver/projectReferences/finding-local-reference-doesnt-load-ancestor/sibling-projects.js

```diff
@@ -265,58 +329,22 @@ FsWatches::
   {}
 /user/username/projects/solution/tsconfig.json:
   {}
+/user/username/projects/solution/services/tsconfig.json:
+  {}
 
 FsWatchesRecursive::
 
-Info 29   [00:01:12.000] Finding references to /user/username/projects/solution/compiler/program.ts position 110 in project /user/username/projects/solution/compiler/tsconfig.json

```

Comment:
These baselines changed in a different way than the rest, but what I think is going on is that before we had an `updateGraphWorker` call being logged inside the call to `Finding references to /user/username/projects/solution/compiler/program.ts position 110 in project /user/username/projects/solution/compiler/tsconfig.json`, but now that update is being done and logged above, in the previous call to find all references, so it doesn't show here because the update happens earlier now.

Reply:
This is example of why you done want to update all projects.. your solution (eg in our case we could be working on checker) can containing many projects but local find all ref should not load other projects (eg local checker function shouldnt load unittests or server or other projects)